fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractIncludeRevincludeTest.java

```diff
@@ -120,8 +122,11 @@ public void createResources() throws Exception {
         savedObservation5 = observation.toBuilder().subject(reference("Patient/" + savedPatient3.getId())).build();
```

Comment:
General comment. Are there any negative tests for bad parameters like `include:iteration`

Reply:
I added multiple parsing tests in InclusionParameterParseTest, including one for invalid modifier.